### PR TITLE
injections: add more regex injections

### PR DESCRIPTION
**Pull request by ObserverOfTime**
I added a missing node to the `string.regex` highlights for Perl but didn't add the injections because the patterns might include expressions which should be interpolated and highlighted by the grammar (though they currently aren't).

### CODE REVIEW

File: queries/julia/injections.scm [hunk 1]
```diff
@@ -1,8 +1,13 @@
-; TODO: re-add when markdown is added.
-; ((triple_string) @markdown
-;   (#offset! @markdown 0 3 0 -3))
+((string_literal) @markdown
+ (#match? @markdown "^\"\"\"")
+ (#offset! @markdown 0 3 0 -3))
```
Comment (theHamsta) [inline queries/julia/injections.scm:3]: This is unrelated to regexes

Comment (ObserverOfTime) [reply queries/julia/injections.scm:3]: True, but I noticed the todo and decided to fix it. I'll split the commit.

Comment (theHamsta) [reply queries/julia/injections.scm:3]: It should probably work fine now

**Review by theHamsta — APPROVED** [review]

File: queries/julia/injections.scm [hunk 2]
```diff
@@ -1,8 +1,13 @@
...
 
 [
   (line_comment)
   (block_comment)
 ] @comment
+
+((prefixed_string_literal
```
Comment (theHamsta) [inline queries/julia/injections.scm:10]: Should regex really be injected into all raw-strings?

Comment (ObserverOfTime) [reply queries/julia/injections.scm:10]: `typeof(r"foo")` is `Regex` according to the [manual](https://docs.julialang.org/en/v1/manual/strings/#man-regex-literals).

Comment (clason) [reply queries/julia/injections.scm:10]: @theHamsta look further below: it only injects into strings prefixed with `r`

Comment (theHamsta) [reply queries/julia/injections.scm:10]: Sorry, for the confusion. I thought it would be similar to Python r-strings
